src/ansys/fluent/core/solver/flobject.py

```diff
@@ -292,6 +292,19 @@ class Real(SettingsBase[RealType], Numerical):
     expression values.
     """
 
+    def value(self):
+        """Get the value of the object."""
+        val = self.get_attr("value", (float, int))
+        return val
+
+    def quantity_map(self) -> Dict[str, float]:
+        """Get the units-quantity of the object."""
+        val = self.get_attr("units-quantity", str)
+        if isinstance(val, str):
+            val = val.replace("'", '')
+            val = ' '.join(word.capitalize() for word in val.split())
```

Comment:
Do we need to convert a symbol-string to PascalCase here (`"'heat-flux"` -> `"Heat Flux"`)?

`val = ' '.join(x.capitalize() for x in val.lstrip("'").split("-"))`

Reply:
We have used PascalCase in `quantity_map_method` because it's also used in CFX units library.

From `quantity_map_method` -

![image](https://user-images.githubusercontent.com/106588300/209148412-5efbd22a-a214-4cd1-afc8-5f5a75f87844.png)

From CFX - 

![image](https://user-images.githubusercontent.com/106588300/209148616-faf983ac-b558-488a-a7db-d4f1c4db3aff.png)

Reply:
Yeah, I got that. But the current code doesn't split `"heat-flux"` by `"-"`.

Reply:
Right. Got it. It's corrected now. Thank you very much.